modules/Bio/EnsEMBL/Analysis/Hive/Config/ShortncRNA.pm

```diff
@@ -0,0 +1,534 @@
+          .' -dbhost #expr(#target_db#->{-host})expr#'
+          .' -dbport #expr(#target_db#->{-port})expr#'
+          .' -dbuser #expr(#target_db#->{-user})expr#'
+          .' -dbpass #expr(#target_db#->{-pass})expr#'
+          .' -dbname #expr(#target_db#->{-dbname})expr#'
+          .' -idfile '.catfile($self->o('ncrna_dir'), 'mirnas_to_delete.txt'),
+        target_db => $self->o('ncrna_db'),
+      },
+      -rc_name   => 'filter',
+      -flow_into => {
+        1 => 'ncrna_sanity_checks',
+      },
+    },
+
+    {
+      -logic_name => 'ncrna_sanity_checks',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveAnalysisSanityCheck',
+      -parameters => {
+        target_db => $self->o('ncrna_db'),
+        sanity_check_type => 'gene_db_checks',
+        min_allowed_feature_counts => get_analysis_settings('Bio::EnsEMBL::Analysis::Hive::Config::SanityChecksStatic',
+            'gene_db_checks')->{$self->o('uniprot_set')}->{'ncrna'},
+      },
+      -rc_name    => '4GB',
+    },
+  ];
+}
+
+
+sub resource_classes {
+  my $self = shift;
+
+  return {
+    'default' => { LSF => $self->lsf_resource_builder('production-rh74', 900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '4GB' => { LSF => $self->lsf_resource_builder('production-rh74', 4000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '5GB' => { LSF => $self->lsf_resource_builder('production-rh74', 5000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '6GB' => { LSF => $self->lsf_resource_builder('production-rh74', 6000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '10GB' => { LSF => $self->lsf_resource_builder('production-rh74', 10000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    'blast' => { LSF => $self->lsf_resource_builder('production-rh74', 2900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], undef, 3)},
+    'blast_retry' => { LSF => $self->lsf_resource_builder('production-rh74', 5900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], undef, 3)},
+    'filter' => { LSF => $self->lsf_resource_builder('production-rh74', 4900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'genblast_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+  }
+}
+
+sub hive_capacity_classes {
+  my $self = shift;
+
+  return {
+    'hc_very_low'    => 35,
+    'hc_low'    => 200,
+    'hc_medium' => 500,
```

Comment:
```suggestion
    'hc_very_low'    => 35,
    'hc_low'    => 200,
    'hc_medium' => 500,
```
If we only keep resource_classes and hive_capacity_classes used in this file these 3 could be removed.